examples/pytorch/logging/pytorch_log_model.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "7066087c-6947-4a50-a7fb-6ab71c004ccf",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Log a pytorch model\n",
+    "\n",
+    "In this guide we will:\n",
+    "- Define a simple pytorch model\n",
+    "- Set a model signature for our logged model to define inputs and outputs to the mlflow model\n",
+    "- Log our model to MLflow server\n",
+    "- Load the model back from storage, to use in other notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "885e8968-cc18-48db-af92-1b034c6ea3af",
+     "showTitle": true,
+     "title": "Install dependencies"
+    },
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%pip install mlflow torch\n",
+    "import warnings\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "092ae65d-cebd-4778-8db5-400097966a3e",
+     "showTitle": true,
+     "title": "Imports + data preparation"
```

Comment:
We can separate the title, package import is not closely related to data preparation.